.github/workflows/release-dataset.yml

```diff
@@ -0,0 +1,41 @@
+name: Release dataset to production DB
+
+on:
+  workflow_dispatch:
+    branches: 
+      - main
+    inputs: 
+      megaset-name:
+        description: "Enter the name of the megaset to release every dataset within"
+        required: false
+      dataset-id: 
+        description: "Enter the ID in the format [NAME]_v[VERSION] to release a single dataset"
+        required: false
+      
+
+jobs: 
+  release_dataset: 
+    name: Release Dataset
+    runs-on: ubuntu-latest
+    env:
+      NODE_ENV: "production"
+      FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
+      FIREBASE_EMAIL: ${{ secrets.FIREBASE_EMAIL }}
+      AWS_SECRET: ${{ secrets.CFE_AWS_SECRET }}
+      AWS_ID: ${{ secrets.CFE_AWS_ID }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+      - run: npm ci
+      - name: Release every dataset in a megaset
+        if: ${{ github.event.inputs.megaset-name == 'true' }} && ${{ github.event.inputs.dataset-id == 'false' }}
```

Comment:
this will never equal true. I think you are trying to check if these exist. So if you wanted to you could set a default to "default" or something, and then check if `${{ github.event.inputs.megaset-name != "default" }}. You're also checking against the string "true", where if you were trying to find out if a value is true, you'd do `${{ github.event.inputs.megaset-name == true }} (no quotes). But this only makes sense if the `type` was set to `boolean` in the inputs

Reply:
ah I saw what you meant, sorry I got confused with "required" and "default". Just pushed new commit, thank you!!